templates/deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
             {{- end }}
             {{- with .Values.shadowsocks.dnsServers }}
             - name: DNS_ADDRS
-              value: {{ join "," . }}
+              value: {{ quote join "," . }}
```

Comment:
You forgot the parentheses here. This will cause the following exception:
```
Error: template: shadowsocks/templates/deployment.yaml:58:30: executing "shadowsocks/templates/deployment.yaml" at <join>: wrong number of args for join: want 2 got 0
```

Reply:
Hi! Thanks for the feedback, I got confuse in the tests, my change wasn't even working.

I force-pushed a fix. To test it please create a `value-test.yaml` with the following content:
```yaml
shadowsocks:
  dnsServers: []
```
; and then run `helm install --dry-run --values values-test.yaml --debug test .`
You should get the following:
```
env:
  - ...
  - name: DNS_ADDRS
    value: ""
```